pkg/vz/vm_darwin.go

```diff
@@ -344,6 +344,16 @@ func attachFolderMounts(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineCo
 		config.SetDirectoryShare(share)
 		mounts[i] = config
 	}
+
+	if driver.Yaml.Rosetta.Enabled {
+		logrus.Info("Setting up Rosetta share")
+		directorySharingDeviceConfig, err := createRosettaDirectoryShareConfiguration(driver.Yaml.Rosetta.Install)
```

Comment:
Just a thought,

Do we needed Rosetta.Install ? 
Can we directly install since Rosetta.Enabled is true already ?

Reply:
I'm not sure. I figured it would make sense to make it configurable in case users want to install Rosetta themselves. I already have it installed, so I'm not sure what it actually looks like if you try without it.

Reply:
During installation nothing will pop-up or happen. It just like installation via command line. So i think it should be fine. 

But let's also hear others opinion on this.

Reply:
> Can we directly install since Rosetta.Enabled is true already ?

:+1:

Reply:
> Can we directly install since Rosetta.Enabled is true already ?

:+1: , assuming that `Enabled = true` doesn't show the macOS password popup as long as Rosetta is already installed on the macOS host.

Reply:
@AkihiroSuda Your assumption is true. Only for installation the password pop-up will be shown. Not for every run of rosetta.

Reply:
PR updated